test-macros/src/lib.rs

```diff
@@ -5,12 +5,11 @@ use syn::{parse_macro_input, ItemFn, Stmt};
 #[proc_macro_attribute]
 pub fn abort_on_panic(_attr: TokenStream, item: TokenStream) -> TokenStream {
     let mut input: syn::ItemFn = parse_macro_input!(item as ItemFn);
-    let test_case = &input.sig.ident.to_string();
 
     let panic_hook: Stmt = syn::parse_quote! {
         std::panic::set_hook(Box::new(|info| {
             let stacktrace = std::backtrace::Backtrace::force_capture();
-            println!("`{}` panic! \n@info:\n{}\n@stackTrace:\n{}", #test_case, info, stacktrace);
+            println!("test panic! \n@info:\n{}\n@stackTrace:\n{}", info, stacktrace);
```

Comment:
Can we parse the test case name from the first stack frame?

Reply:
the test case name is irregularity in backtrace, we can't parse the test names unless we force all test names to conform to the specified format